src/components/SearchResultTags.js

```diff
@@ -317,10 +324,15 @@ class SearchResultTags extends React.Component<SearchResultTagsDefaultProps, Sea
       </span>
     ) : '';
 
+    const comments = this.props.comments ? (
```

Comment:
This could just be: 
```
    const comments = this.props.comments && (
      <Comments docId={this.props.docId} commentsTable={this.props.commentsTable} />
    );
```